src/main/java/nl/eveoh/mytimetable/blackboard/service/PersonalizationService.java

```diff
@@ -0,0 +1,53 @@
+package nl.eveoh.mytimetable.blackboard.service;
+
+import blackboard.data.ValidationException;
+import blackboard.persist.PersistenceException;
+import blackboard.portal.external.CustomData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PersonalizationService extends HttpServlet {
+
+    private static final Logger log = LoggerFactory.getLogger(PersonalizationService.class);
+
+    public static final String EDIT_JSP = "edit.jsp";
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+        CustomData cd;
+        String numberOfActivities = req.getParameter("numberOfActivities");
+
+        Map<String, String> messages = new HashMap<String, String>();
+
+        try {
+            cd = CustomData.getModulePersonalizationData(req);
+            cd.setValue("numberOfActivities", numberOfActivities);
```

Comment:
Hmmjah, kunnen we in ieder geval checken dat dat ding meer is dan 0, en bij voorkeur ook minder is dan de max, voordat we het opslaan?